packages/atomic/dev/examples/commerce-website/commerce-nav.mjs

```diff
@@ -16,6 +16,11 @@ export const navContent = {
     label: 'Towels',
     barcaUrl: `${barcaRoot}/browse/promotions/accessories/towels`,
   },
+  Toys: {
```

Comment:
I added this to manually test the fix in a listing page (the other pages have too few products to have a load more button).